developer/src/kmc/test/test-infrastructureMessages.ts

```diff
@@ -72,6 +155,17 @@ describe('InfrastructureMessages', function () {
       `ERROR_FileTypeNotFound not generated, instead got: `+JSON.stringify(ncb.messages,null,2));
   });
 
+  // ERROR_CannotCreateFolder
+
+  it('should generate ERROR_CannotCreateFolder if the output folder cannot be created', async function() {
+    const buildKeyboardInfo = new BuildKeyboardInfo();
+    const targetFilename = makePathToFixture('invalid-projects', 'build', 'error_cannot create_folder.xxx')
```

Comment:
```suggestion
    const targetFilename = makePathToFixture('invalid-projects', 'build', 'error_cannot_create_folder.xxx')
```

This seems like it may not have actually been testing what you expected if the fixture had the wrong name?

Reply:
Correct missing underscore in filename. 

Because it strips off the filename - it just needs the folder name - the filename does not affect the test.  What matters is that there is a file (build) where the test tries to create a folder.